### ISSUE TITLE
Dar feedback na avaliação 

### ISSUE THREAD

**Issue by gmatosferreira** [rt-evil-inc/gira-mais#12]
Quando avaliamos uma viagem com 3 ou menos é-nos permitido dar feedback sobre a mesma na aplicação da EMEL. Penso que é importante esta feature, por permite reportar problemas com as bicicletas.

**Comment by ttmx** [comment]
Está planeado! A versão atual é uma simplificação necessária devido a restrições de tempo.

**Comment by rodrigohpalmeirim** [comment]
Sim, acho que ficou fixe! Foi a Inês Freitas que fez (ela é designer) :)
Deixei aqui porque um rapaz me pediu para arranjar alguma coisa em que contribuir para o projeto!
